test/Microsoft.TemplateEngine.Orchestrator.RunnableProjects.UnitTests/TemplateConfigTests/ConstraintsTest.cs

```diff
@@ -62,7 +62,7 @@ public void CanReadConstraintDefinition()
 
             Assert.Equal("\"arg\"", model.Constraints[0].Args);
             Assert.Equal("[\"one\",\"two\",\"three\"]", model.Constraints[1].Args);
-            Assert.Equal("{\"one\":\"one\",\"two\":\"two\"}", model.Constraints[2].Args);
+            Assert.Equal(/*lang=json,strict*/ "{\"one\":\"one\",\"two\":\"two\"}", model.Constraints[2].Args);
```

Comment:
The raw string syntax also works for single-line strings. I'm assuming this can be combined with the lang=json syntax.

```suggestion
            Assert.Equal(/*lang=json,strict*/ """{"one":"one","two":"two"}""", model.Constraints[2].Args);
```